.lintr

```diff
@@ -7,4 +7,4 @@ linters: with_defaults(
   single_quotes_linter = NULL,
   trailing_blank_lines_linter = NULL
   )
-exclusions: list("R/hexamers.R", "inst/doc/intro.R")
+exclusions: list("R/hexamers.R", "inst/doc/intro.html")
```

Comment:
Why did this change? Was this giving you issue? The linter shouldn't be checking anything other than .R files